### PR TITLE
Comp Bio Guide

### PR DESCRIPTION
**Pull request by Zmwang622**
Also removed Stats major guide because that's no longer being written

### CODE REVIEW

File: src/guides/academics/majors/stats.md [hunk 1]
```diff
@@ -1,4 +1,6 @@
 ---
-title: "Stats Major"
-author: "Alex Zharichenko"
+title: "Stats Guide"
+author: "Alex Z"
```
Comment (varughese) [inline src/guides/academics/majors/stats.md:3]: Why not just delete this?

Comment (Zmwang622) [reply src/guides/academics/majors/stats.md:3]: i deleted it and the build failed

**Comment by varughese** [comment]
Prob cuz it's linked in academics/index.md

**Comment by Zmwang622** [comment]
yeah fixing now

**Comment by Zmwang622** [comment]
What would the compbio link be? `/academics/major/comp-bio`?

**Comment by varughese** [comment]
Yeah, The path to the file

**Comment by Zmwang622** [comment]
did it , approve?

**Pull request merged**